L1Trigger/TrackFindingTracklet/src/VMRouterTable.cc

```diff
@@ -63,10 +81,12 @@ void VMRouterTable::init(unsigned int layerdisk, std::string const& name) {
       if (layerdisk >= N_LAYER) {
         double rproj = r * settings_.zmean(layerdisk - N_LAYER) / z;
         bin = 0.5 * NBINS * (rproj - settings_.rmindiskvm()) / (settings_.rmaxdiskvm() - settings_.rmindiskvm());
+        //bin value of zero indicates that stub is out of range
         if (bin < 0)
           bin = 0;
         if (bin >= NBINS / 2)
-          bin = NBINS / 2 - 1;
+          //bin = NBINS / 2 - 1;
```

Comment:
what's going on here? should this out-commented be deleted or is it a temporary fix?

Reply:
Should be deleted - now done.